sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -423,6 +426,102 @@ case class ShowTablePropertiesCommand(
 }
 
 /**
+ * A command for users to list the column names for a table. This function creates a
+ * [[ShowColumnsCommand]] logical plan.
+ *
+ * The syntax of using this command in SQL is:
+ * {{{
+ *   SHOW COLUMNS (FROM | IN) table_identifier [(FROM | IN) database];
+ * }}}
```

Comment:
@liancheng I followed other commands and some of them mention the SQL syntax. Do you want them removed from all other commands or just this two ?